src/shared_utilities/Domains.jl

```diff
@@ -357,6 +372,10 @@ function HybridBox(;
 
     surface_space = obtain_surface_space(subsurface_space)
     space = (; surface = surface_space, subsurface = subsurface_space)
+    z = ClimaCore.Fields.coordinate_field(subsurface_space).z
+    Δz_top, Δz_bottom = get_Δz(z)
+    z_sfc = get_z_sfc(z)
+    fields = (; z = z, Δz_top = Δz_top, Δz_bottom = Δz_bottom, z_sfc = z_sfc)
```

Comment:
Since we do the exact same thing for each domain type, it might be nice to pull this out into a helper function, e.g. `get_domain_fields(subsurface_space)`. Then it will be much easier if we want to change the fields we store here

Reply:
great idea! I did this